spec/fixtures/listens.yml

```diff
@@ -0,0 +1,8 @@
+valid_listen:
+  listener: sudara
+  asset: valid_mp3
+  track_owner: arthur
+valid_listen:
```

Comment:
You can't have multiple records with the same key in the fixtures. I generally like to use the key do describe what it does or why it's there. So in this case maybe `arthur_listens_to_valid_mp3`.

Reply:
oops. copy/pasta